spec/index.html

```diff
@@ -3167,6 +3167,16 @@ <h5>The <code>&lt;<dfn id="the-coordinates-element">coordinates</dfn>&gt;</code>
         </p>
         </section>
         </section>
+        
+        <section class="informative">
+        <h3>Authoring</h3>
+        <p>
+        While MapML is human-readable and can be written by hand,
+        authors typically will use conversion programs
+        and other specialized software tools to generate MapML.
```

Comment:
I believe "authors" in this context can mean both the application author and map data provider. Should this be clarified? If so, how exactly?

Reply:
Looks good to me.  The equivalent in mapping are servers like GeoServer and MapServer, which serve images and documents according to standard APIs, like WMS, WFS etc.  I thought I had mentioned them somewhere, but I agree it should be up front.  Entering coordinates by hand is daunting for simple image maps; much more so for actual geographic features and so on.